### PR TITLE
Add suggestions from post-merge review feedback

### PR DESCRIPTION
**Pull request by cvializ**
Addressing some of the feedback from @jridgewell in #10283

### CODE REVIEW

File: extensions/amp-analytics/0.1/events.js [hunk 1]
```diff
@@ -438,41 +438,39 @@ export class VideoEventTracker extends EventTracker {
...
+      }
+
+      if (excludeAutoplay && details['state'] === PlayingStates.PLAYING_AUTO) {
+        return;
+      }
+
+      const el = dev().assertElement(event.target,
+          'No target specified by video session event.');
+      targetReady.then(target => {
+        if (target.contains(el)) {
```
Comment (aghassemi) [inline extensions/amp-analytics/0.1/events.js]: Hmm, I might be understanding this wrong, but can't `target` and `el` be the same? video-manager dispatches events on the element itself (e.g. `<amp-video id="foo">` is target) now if analytics spec is set as `select="#foo"`, wouldn't this check fail?

Comment (cvializ) [inline extensions/amp-analytics/0.1/events.js]: They can be the same, but `Element#contains` is true if you pass the same element.

File: src/service/video-manager-impl.js [hunk 2]
```diff
@@ -376,10 +376,14 @@ class VideoEntry {
    * @private
    */
   videoPaused_() {
-    const trackingVideo = assertTrackingVideo(this.video);
-    if (trackingVideo &&
-        trackingVideo.getCurrentTime() !== trackingVideo.getDuration()) {
-      this.analyticsEvent_(VideoAnalyticsType.PAUSE);
+    const v = this.video;
+    if (isTrackingVideo(v)) {
+      const trackingVideo =
```
Comment (jridgewell) [inline src/service/video-manager-impl.js]: Why is the type cast needed? How are `#getCurrentTime` and `#getDuration` not implemented for a normal video?

Comment (cvializ) [inline src/service/video-manager-impl.js]: I was going to remove the extra interface in a future PR but I suppose I can knock it out it in this one

Comment (cvializ) [inline src/service/video-manager-impl.js]: Done.

**Comment by aghassemi** [comment]
@cvializ see https://github.com/ampproject/amphtml/pull/10435, please unskip them in this PR.

File: extensions/amp-analytics/0.1/events.js (hunk 1 above)
Comment (aghassemi) [inline extensions/amp-analytics/0.1/events.js]: TIL... :)

**Comment by cvializ** [comment]
PTAL when you get a chance

**Pull request merged**